tests/test_config_parsing/test_parser_error_collection.py

```diff
@@ -901,3 +925,146 @@ def test_that_executable_directory_errors(dirname):
             end_column=12 + len(dirname),
         ),
     )
+
+
+@pytest.mark.parametrize(
+    "contents, expected_errors",
+    [
+        (
+            """
+NUM_REALIZATIONS  1
+RUNPATH %d
+
+RELPERM
+MULTZ Hehe 2 l 1 2
+EQUIL Hehe 2 l 1 2
+GEN_PARAM Hehe 2 l 1 2
+MULTFLT Hehe 2 l 1 2
+
+UMASK 1 2 3 4 111
+LOG_FILE 1 2 3 4 111
+LOG_LEVEL 1 2 3 4 111
+ENKF_RERUN 1 2 3 4 111
+
+RSH_HOST boq
+RSH_COMMAND qqqwe
+MAX_RUNNING_RSH assadsa 3 12w qwsa
+
+LSF_SERVER
+LSF_QUEUE
+MAX_RUNNING_LSF
+MAX_RUNNING_LOCAL
+
+SCHEDULE_PREDICTION_FILE
+HAVANA_FAULT
+REFCASE_LIST
+RFTPATH
+END_DATE
+CASE_TABLE
+RERUN_START
+DELETE_RUNPATH
+PLOT_SETTINGS
+UPDATE_PATH
```

Comment:
I think you break your own test when removing the arguments, UPDATE_PATH require two arguments.